src/plugins/visualizations/public/embeddable/disabled_lab_visualization.tsx

```diff
@@ -18,28 +18,30 @@
  */
 
 import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiCallOut } from '@elastic/eui';
 import React from 'react';
 
+import './disabled_lab_visualization.scss';
+
 export function DisabledLabVisualization({ title }: { title: string }) {
+  const calloutTitle = (
+    <FormattedMessage
+      id="visualizations.disabledLabVisualizationTitle"
+      defaultMessage="{title} is a lab visualization."
+      values={{ title }}
+    />
+  );
   return (
-    <div className="visDisabledLabVisualization">
-      <div
-        className="kuiVerticalRhythm visDisabledLabVisualization__icon kuiIcon fa-flask"
-        aria-hidden="true"
+    <EuiCallOut
+      className="visDisabledLabVisualization"
+      size="s"
+      title={calloutTitle}
+      iconType="beaker"
+    >
+      <FormattedMessage
+        id="visualizations.disabledLabVisualizationMessage"
+        defaultMessage="Please turn on lab-mode in the advanced settings to see lab visualizations."
       />
-      <div className="kuiVerticalRhythm">
-        <FormattedMessage
-          id="visualizations.disabledLabVisualizationTitle"
-          defaultMessage="{title} is a lab visualization."
-          values={{ title: <em className="visDisabledLabVisualization__title">{title}</em> }}
-        />
-      </div>
-      <div className="kuiVerticalRhythm">
-        <FormattedMessage
-          id="visualizations.disabledLabVisualizationMessage"
-          defaultMessage="Please turn on lab-mode in the advanced settings to see lab visualizations."
-        />
-      </div>
-    </div>
+    </EuiCallOut>
```

Comment:
We should make sure to check permissions of accessing advanced settings for whether we want to display that link. Otherwise empty prompt sounds like a good solution.

Reply:
After syncing with @MichaelMarcialis I added a link to advanced settings docs https://www.elastic.co/guide/en/kibana/current//advanced-options.html#kibana-visualization-settings

<img width="791" alt="Screenshot 2020-11-09 at 4 48 01 PM" src="https://user-images.githubusercontent.com/17003240/98556483-c385a600-22ab-11eb-8a7f-eaa2a91c8b60.png">